wdl/PETestChromosome.wdl

```diff
@@ -201,22 +201,28 @@ task PETest {
     sort -k1,1 -k2,2n region.bed > region.sorted.bed
     bedtools merge -d 16384 -i region.sorted.bed > region.merged.bed
 
-    # Temporary workaround for corrupted tabix downloads
-    x=0
-    while [ $x -lt ~{tabix_retries} ]
-    do
-      # Download twice
-      GCS_OAUTH_TOKEN=`gcloud auth application-default print-access-token` tabix -R region.merged.bed ~{discfile} | bgzip -c > PE_1.txt.gz
-      GCS_OAUTH_TOKEN=`gcloud auth application-default print-access-token` tabix -R region.merged.bed ~{discfile} | bgzip -c > PE_2.txt.gz
-      # Done if the downloads were identical, otherwise retry
-      cmp --silent PE_1.txt.gz PE_2.txt.gz && break       
-      x=$(( $x + 1))
-    done
-    echo "PE-tabix retry:" $x
-    cmp --silent PE_1.txt.gz PE_2.txt.gz || exit 1
+    if [ -s region.merged.bed ]; then
```

Comment:
For future reference, which part of this was failing in empty shards?

Reply:
The thing I'm trying to fix with this change cropped up after I made the change to the `SplitVCF` task in `Tasks02.wdl`. That task can now produce an empty VCF file (with header, but no records in it). When there aren't any records in the input VCF to this task, the awk and bedtools commands above end up producing an empty `region.merged.bed` file. When you give tabix an empty bed file, it interprets that as a lack of intervals, so the remote tabix command I wrapped here would try to localize the entire PE or SR file. This avoids that; afterwards `svtk pe-test` operates gracefully on the empty VCF shard, returning an empty stats table.

Reply:
Ok thanks. We did run into a problem like this, but it was only for the common regions, which motivated the changes to PE/SRTest in the other PR. The new GATK tool also will localize the whole remote file if the intervals list is empty, so I'll just merge these changes manually to include your empty file check.